### ISSUE TITLE
In api/windowscounters, add a way to query multiple counters in one API call

### ISSUE THREAD

**Issue by Madlohe** [NagiosEnterprises/ncpa#506]
I don't know how necessary this is, but for hosts that have a lot of performance counters to monitor (especially ones that also require &sleep=1), it might be beneficial to nagios/NCPA's performance to have multiple win32pdh counters on one query

**Comment by petrolej** [comment]
Yes, I would appreciate this. At least to have an option to get counters for all instances - for example all disks or all cpu cores or all network interfaces etc.

**Comment by sawolf** [comment]
@petrolej - It's been a long time since I messed with this, but I think there are ways you can do this just using a "single" performance counter - you would just need to set the instance to * or _Total. If you're using Nagios XI, there are some examples of this in the checks we create from the Hyper-V configuration wizard.

**Comment by petrolej** [comment]
Thank you, for your answer @sawolf . I have already tried that but unfortunately this is not the way it works. The command does not result in error but it returns (IMHO) only the first instance.

Example:
`/usr/local/nagios/libexec/check_ncpa.py -H 10.10.10.10 -t password -M "windowscounters/PhysicalDisk(*)/Avg. Disk Queue Length" -q "sleep=1" -T 59 -w 2 -c 5`

Return only:
`OK: \PhysicalDisk(*****)\Avg. Disk Queue Length was 0.01 Avg. Disk Queue Length | '\PhysicalDisk(*****)\Avg. Disk Queue Length'=0.01;2;5;`

But it should be something like:
`OK: \PhysicalDisk(**0 C:**)\Avg. Disk Queue Length was 0.01 Avg. Disk Queue Length, \PhysicalDisk(**0 D:**)\Avg. Disk Queue Length was 0.01 Avg. Disk Queue Length | '\PhysicalDisk(**0 C:**)\Avg. Disk Queue Length'=0.01;2;5;` '\PhysicalDisk(**0 D:**)\Avg. Disk Queue Length'=0.01;2;5;`

I already ended up with a simple script which performs both tests (if only two drives) and creates the output text in the way above.

The way with giving it a "*" symbol would be great and I think it is not a difficult to code it. The NCPA should just enumerate all the instances and compose the output with all of them.

Best regards
Petr
